### PR TITLE
Vulpkanin Update

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Updates vulpkanin to give them extra language features
wyY8lrmeNI

This uses the magic of RustG to make this extremely efficient code.

*no really someone made this shit SSE4.1 accelerated. The fact someone made a hardware-accelerated text-uwuification engine is really a mark to show the point humanity is at*

Intel's website says the 9900K supports this instruction so we should be fine.

## Why It's Good For The Game
Gotta appeal to the fandom

### CODE REVIEW

File: code/__DEFINES/rust_g.dm [hunk 1]
```diff
@@ -6,6 +6,9 @@
 // Gets the version of RUSTG
 /proc/rustg_get_version() return call(RUST_G, "get_version")()
 
+// uwuifies text
+#define rustg_uwuify(text) call(RUST_G, "uwuify")(text)
```
Comment (Edan52) [inline code/__DEFINES/rust_g.dm:10]: wustg

Comment (AffectedArc07) [reply code/__DEFINES/rust_g.dm:10]: GOOD FUCKING POINT

Comment (AffectedArc07) [inline code/__DEFINES/rust_g.dm:10]:
```suggestion
#define wustg_uwuify(text) call(RUST_G, "uwuify")(text)
```

Doing it this way means I can webedit

File: code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -136,6 +136,9 @@ GLOBAL_LIST_EMPTY(channel_to_radio_key)
 
 	message = trim_left(message)
 
+	if(isvulpkanin(src))
+		message = rustg_uwuify(message) // Gotta get those accuracies in
```
Comment (AffectedArc07) [inline code/modules/mob/living/say.dm:140]:
```suggestion
		message = wustg_uwuify(message) // Gotta get those accuracies in
```

**Comment by dearmochi** [comment]
uwu

**Comment by AzuleUtama** [comment]
https://www.youtube.com/watch?v=0RFwyobtnKA

**Comment by Spacemanspark** [comment]
the update they all deserve

**Comment by Mitchs98** [comment]
Merge this immediately.

**Comment by KasparoVy** [comment]
Woof

**Comment by procdrone** [comment]
Time to play rawrfox again i see

**Comment by meow20** [comment]
I thought its already in the game.

**Comment by AffectedArc07** [comment]
Selfmerging is a god given right

**Pull request merged**